src/auth.ts

```diff
@@ -152,6 +167,18 @@ export const config = async (request: NextRequest | undefined) => {
         profile: getDiscordProfile(adapterUser),
       }),
       GitHub,
+      Google({
+        authorization: {
+          params: {
+            // https://github.com/nextauthjs/next-auth/blob/748c9ecb8ce10bef2b628520451f676db0499f9d/docs/pages/guides/configuring-oauth-providers.mdx
+            scope: 'openid https://www.googleapis.com/auth/calendar',
+            // https://next-auth.js.org/providers/google
```

Comment:
v5 のドキュメントこっちなのよ
まじでわかりにくいよな
```suggestion
            // https://authjs.dev/getting-started/providers/google
```